target/linux/rockchip/config-default

```diff
@@ -1,297 +0,0 @@
-CONFIG_64BIT=y
```

Comment:
I don’t think it’s a good idea to move config-default to config-5.x, we should try to be consistent with upstream as much as possible.

Reply:
First, it's actually made by upstream.
Second, there're some differences between 5.4 and 5.10, so it's the right approach.